Firebase/Auth/README.md

```diff
@@ -6,3 +6,107 @@ for their end users.
 Please visit [our developer site](https://firebase.google.com/docs/auth/) for
 integration instructions, documentation, support information, and terms of
 service.
+
+# Firebase Auth Development
+
+Example/Auth contains a set of samples and tests that integrate with
+FirebaseAuth.
+
+The Podfile specifies the dependencies and is used to construct an Xcode
+workspace consisting of the samples, modifiable FirebaseAuth library, and its
+dependencies.
+
+
+### Running Sample Application
```

Comment:
The detailed instructions below are directly related to the sample app. Does it make sense to put these into the sample app folder?

Reply:
+1, it would be sort of confusing to leave these here.

Reply:
Moved. Thanks.